### ISSUE TITLE
Add custom data and augment the RIPE data with that

### ISSUE THREAD

**Issue by brunoerg** [brunoerg/asmapy#7]
cc: @dunxen

**Comment by dunxen** [comment]
Thanks! Yeah so in addition to the RIPE data, we'd also want users to be able to supply their own router dump for creating maps (maybe via some CLI argument to the appropriate command(s)).

**Comment by brunoerg** [comment]
Nice! 

Questions:
1. User has its own router dump in what format? Same format when we download RIPE data, so we want to merge them?
2. User wants to supply their whole router dump or only part of that?

**Comment by dunxen** [comment]
Yes, I was thinking of just being able to merge their whole dump (same format as RIPE) and using some strategy to resolve conflicts (maybe preferring the custom dump). If that makes sense?

**Comment by brunoerg** [comment]
Yes, make sense! I'm gonna work on it!

**Comment by brunoerg** [comment]
@Mistersx12 Good example, but you're not considering both files are in the same format I guess. But it can be a good start! :)
